content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -299,6 +299,11 @@ Expanding EBS volumes is a time-consuming operation. Also, there is a per-volume
 
 #### Recovering from Failure when Expanding Volumes
 
+If a user specifies a new size that is too big to be satisfied by underlying storage system, expansion of PVC will be continuously retried until user or cluster administrator takes some action. This can be undesirable and hence  Kubernetes provides following methods of recovering from such failures.
```

Comment:
nit: extra white spaces between "hence" and "Kubernetes".

Reply:
fixed